tools/capacity-planner-cli/README.md

```diff
@@ -0,0 +1,163 @@
+# Capacity Planner CLI
+
+This tool is a stand-alone tool to extract peak resource usage values and corresponding timestamps for a given GCP project, time range and timezone.
+
+
+## Usage
+
+You should run the following command to acquire the Application Default Credentials (ADC). Alternatively you may want to configure `GOOGLE_APPLICATION_CREDENTIALS` environment variable to use your service accounts to avoid hitting lower quota with ADC.
+
+```
+gcloud auth application-default login
+```
+
+```
+% pip install -r requirements.txt
+% python capacity_planner.py --help
+Usage: capacity_planner.py [OPTIONS]
+
+Options:
+  --project_id TEXT               GCP project ID where the Cloud Monitoring
+                                  API is called against.  [required]
+  --end_time [%Y-%m-%dT%H:%M:%S%z]
+                                  The end time in ISO 8601 format of the time
+                                  interval for which           results should
+                                  be returned. Default is now.           e.g.
+                                  2022-10-03T05:23:02+09:00
+  --duration_minutes INTEGER      The number of minutes in the time interval
+                                  ending at the time           specified with
+                                  --end_time. Default is 360 minutes (6
+                                  hours).
+  --output FILE                   The CSV file path for writing the results
+                                  out.           Default is logs/result.csv
+  --help                          Show this message and exit.
+```
+
+
+## Example
+
+The following command will produce a CSV file in `logs/result.csv`.
+
+```
+python capacity_planner.py --project_id my-awesome-project --end_time '2022-10-15T18:30:00+09:00' --duration_minutes 60
+```
+
+
+| product_name<br>NaN    | metric_name<br>NaN            | metrics<br>resource.region | <br>unit | <br>resource.location | value<br>NaN | time<br>NaN               |
```

Comment:
When running this on my end, I noticed that the first row always contains NaN values, any idea why I would be getting that? e.g.
```
python capacity_planner.py --project_id gsk-workflows-pipeline-demo --duration_minutes=10000

```

```
,product_name,metric_name,metrics,metrics,value,time
,nan,nan,resource.location,unit,nan,nan
0,Cloud Storage,QPS,eu,1/s,0.019999777774074013,2023-01-26 12:20:36+00:00
1,Cloud Storage,QPS,us,1/s,0.04999944443518503,2023-01-26 10:20:36+00:00
0,Cloud Storage,Egress MiB/s,eu,MiBy/s,0.15225682833681153,2023-01-26 12:57:36+00:00
1,Cloud Storage,Egress MiB/s,us,MiBy/s,0.020324938955993504,2023-01-26 10:20:36+00:00
```

Reply:
Changed to exclude the index column. Thanks, it is really a good catch.